main.js

```diff
@@ -1,12 +1,205 @@
+}
+
+/**
+ *
+ */
+function attack() {
+    console.log(+'' + 'Figth ...');
+}
+
+/**
+ *
+ * @param name
+ * @returns {HTMLAnchorElement | HTMLElement | HTMLAreaElement | HTMLAudioElement | HTMLBaseElement | HTMLQuoteElement | HTMLBodyElement | HTMLBRElement | HTMLButtonElement | HTMLCanvasElement | HTMLTableCaptionElement | HTMLTableColElement | HTMLDataElement | HTMLDataListElement | HTMLModElement | HTMLDetailsElement | HTMLDialogElement | HTMLDivElement | HTMLDListElement | HTMLEmbedElement | HTMLFieldSetElement | HTMLFormElement | HTMLHeadingElement | HTMLHeadElement | HTMLHRElement | HTMLHtmlElement | HTMLIFrameElement | HTMLImageElement | HTMLInputElement | HTMLLabelElement | HTMLLegendElement | HTMLLIElement | HTMLLinkElement | HTMLMapElement | HTMLMenuElement | HTMLMetaElement | HTMLMeterElement | HTMLObjectElement | HTMLOListElement | HTMLOptGroupElement | HTMLOptionElement | HTMLOutputElement | HTMLParagraphElement | HTMLPictureElement | HTMLPreElement | HTMLProgressElement | HTMLScriptElement | HTMLSelectElement | HTMLSlotElement | HTMLSourceElement | HTMLSpanElement | HTMLStyleElement | HTMLTableElement | HTMLTableSectionElement | HTMLTableCellElement | HTMLTemplateElement | HTMLTextAreaElement | HTMLTimeElement | HTMLTitleElement | HTMLTableRowElement | HTMLTrackElement | HTMLUListElement | HTMLVideoElement}
+ */
+function playerLose(name) {
+    const $loseTitle = createElement('div', 'loseTitle');
+    $loseTitle.innerText = name + ' Lose!!';
+
+    return $loseTitle;
+}
+
+function playerWin(name) {
+    const $wineTitle = createElement('div', 'loseTitle');
+
+    if (name) {
+        $wineTitle.innerText = name + 'Win!!';
+    } else {
+        $wineTitle.innerText = name + 'drow!';
+    }
+    return $wineTitle;
+}
+
+/**
+ * фукция изминения ХП игрока
+ * @param player
+ */
+function changeHP(player) {
+
+    const $playerLife = document.querySelector('.player' + player.player + ' .life');
+
+    $playerLife.style.width = player.hp + '%';
+
+    if (player.hp <= 0) {
+        player.hp = 0;
+    }
+}
+
+function elHP() {
+
+}
+
+function renderHP() {
+
+}
+
+/**
+ * фукция случайных чисел,
+ * @param params
+ * @returns {number}
+ */
+function getRendom (params){
+    const rendom = Math.ceil(Math.random() * params);
+    return rendom;
+}
+
+function createReloadButton() {
+    const $reloadButtonDiv = createElement('div', 'reloadWrap');
+    const $reloadButton = createElement('button', 'button');
+    $reloadButton.innerText = 'Reload';
+
+    $reloadButton.addEventListener('click', function () {
+        window.location.reload();
+    })
+
+    $reloadButtonDiv.appendChild($reloadButton);
+    $root.appendChild($reloadButtonDiv);
+}
+
+function createElement(tag, className) {
+    const $tag = document.createElement(tag);
+    if (className) {
+        $tag.classList.add(className);
+    }
+    return $tag;
+}
+
+function createPlayer(playerObj) {
+    const $player1 = createElement('div', 'player' + playerObj.player);
```

Comment:
давай чтобы не путаться 

```suggestion
    const $player = createElement('div', 'player' + playerObj.player);
```